scripts/calculateGasPrice.js

```diff
@@ -0,0 +1,17 @@
+const hre = require("hardhat");
+
+async function calculateGasPrice() {
+  try {
+    let gasPrice = await hre.ethers.provider.getGasPrice();
+    console.log("Current gasPrice:", gasPrice.toString());
+    gasPrice = gasPrice.mul(120).div(100); // 20% more than the current gas price 
```

Comment:
@hrishikeshio by default setting 20% higher gas price here, should this be made configurable as well? Most chains might not need a higher gas price

Reply:
i dont know. test it on testnets.

Reply:
On all the testnets we did not face these issues actually, the deployments all went smoothly with the existing scripts. I think a 20% higher gasPrice should suffice for all networks.

Reply:
didnt work for opbnb ```[ResultManager] Granting FORWARDER_ROLE to forwarder contract address
√ [hardhat-ledger] Waiting for confirmation
ProviderError: transaction underpriced
    at HttpProvider.request (C:\Users\Hrishikesh\razor\bridge-proxy\node_modules\hardhat\src\internal\core\providers\http.ts:88:21)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at LedgerProvider.request (C:\Users\Hrishikesh\razor\bridge-proxy\node_modules\@nomicfoundation\hardhat-ledger\src\provider.ts:137:18)
    at EthersProviderWrapper.send (C:\Users\Hrishikesh\razor\bridge-proxy\node_modules\@nomiclabs\hardhat-ethers\src\internal\ethers-provider-wrapper.ts:13:20)```